File: DheerajKumar/problem02.cpp
//2. WAP to take input 1 number from the user and print its square and cube.
 #include <iostream>
 using namespace std;

int square(int num){
    return num*num;
}

 int cube(int num){
    return square(num) * num;
 }
 
 int main(){
    int number = 0;
    cout<<"Enter number -> ";
    cin>>number;
    cout<<"\n The square is -> "<<square(number);
    cout<<"\n The cube is -> "<< cube(number);
 }

File: SurajKumarSingh/problem02.java
//2. WAP to take input 1 number from the user and print its square and cube.

import java.util.Scanner;
public class problem02 {
    public static int square(int n){
        return n*n;
    }
    public static int cube(int n){
        return square(n) * n;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int n = sc.nextInt();
        System.out.println("Square of number: " + square(n));
        System.out.println("cube of number: " + cube(n));
        sc.close();
    }
    
}
